Meta/Lagom/Fuzzers/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ add_simple_fuzzer(FuzzUTF16BEDecoder LibTextCodec)
 add_simple_fuzzer(FuzzRSAKeyParsing LibCrypto)
 add_simple_fuzzer(FuzzVP9Decoder LibVideo)
 add_simple_fuzzer(FuzzWAVLoader LibAudio)
+add_simple_fuzzer(FuzzWebPLoader LibGfx)
```

Comment:
Wasm < WebP :^)

Reply:
Done.

In ASCII order, WOFF < Wasm too – want me to move that up? What's our case folding policy for sorting cmake files? (`:sort` in vim would sort it up.)

Reply:
No strong feelings, I think more files use case-insensitive ordering but it's definitely not fully consistent.

Reply:
Let's keep that as-is for now then :)